OpenRA.Mods.Common/Activities/Resupply.cs

```diff
@@ -127,7 +127,7 @@ public override bool Tick(Actor self)
 				// HACK: Repairable needs the actor to move to host center.
```

Comment:
I'm not sure how appropriate this hack label still is now that we don't rely on `ignoreActor` anymore?

Reply:
The hack is that its forcing a move to the center of the actor rather than a non-existant dock, or at least an `Exit` as a workaround. So this should stay for now.